processor/configuration/src/main/java/com/redhat/service/smartevents/processor/sources/SourceResolver.java

```diff
@@ -43,4 +43,14 @@ private String getBridgeWebhookUrl(String customerId, String bridgeId) {
             throw new GatewayProviderException("Can't find events webhook for bridge " + bridgeId);
         }
     }
+
+    @Override
+    public String getType() {
+        return null;
```

Comment:
@lucamolteni This feels like a potential source of NPE. Does `GatewayResolver` need to extend `GatewayBean` anymore seeing as there is currently only one implementation. That would solve our problem here.

Reply:
not having this class implements `GatewayConfiguratorImplTest` will break the `GatewayConfiguratorImplTest` that is tightly coupling the source and the action. It expects a symmetric structure

Reply:
Please don't break the interface hierarchy.
This won't trigger any NPE since it's used only in the `accept` method, which is also overridden to return `true` always. If you fear the NPE put a dummy value but again *please don't break the interface hierarchy*.